prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -288,17 +290,19 @@ class FHIRRouter(
 
     /**
      * Applies all filters to the list of all receivers with topic FULL_ELR that are not set as INACTIVE.
-     * FHIRPath expressions are run against the [bundle] to determine if the receiver should get this message\
+     * FHIRPath expressions are run against the [bundle] to determine if the receiver should get this message
+     * @param bundle
+     * @param report
```

Comment:
Some feedback I got from Carlos on another PR is that param descriptions should all be inline with the function description, like what is already there for `[bundle]`. I think that means `report` should be documented the same way, and we don't need `@param bundle` or `@param report`.

Reply:
Fixed